pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
 spec:
   description: >-
     Tekton pipeline to release content to registry.redhat.io registry
+  timeouts:
```

Comment:
I believe these are valid on a `kind: PipelineRun`, not `kind: Pipeline`

Reply:
You can set it per task in this yaml though like this https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#configuring-the-failure-timeout

Reply:
Hm, I'm afraid it won't work as we expect if we set it on the task.

Will the 1h timeout on the pipelinerun ignore our 2h timeout on the task, if we set it on the task?

Reply:
Updated in c1553f0 to try it.

Reply:
I'm not sure